docs/build/sdks/js/accounts.md

```diff
@@ -9,537 +9,2417 @@ head:
+
+const WITHDRAWAL_HASH = "<WITHDRAWAL_TX_HASH>";
+const isFinalized = await wallet.isWithdrawalFinalized(WITHDRAWAL_HASH);
+```
+
+### `finalizeWithdrawalParams`
+
+Returns the [parameters](./types.md#finalizewithdrawalparams) required for finalizing a withdrawal from the withdrawal transaction's log on the L1 network.
+
+#### Inputs
+
+| Name             | Type        | Description                                                                                                                                         |
+| ---------------- | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `withdrawalHash` | `BytesLike` | Hash of the L2 transaction where the withdrawal was initiated.                                                                                      |
+| `index?`         | `number`    | In case there were multiple withdrawals in one transaction, you may pass an index of the withdrawal you want to finalize. Defaults to 0 (optional). |
+
+```ts
+async finalizeWithdrawalParams(withdrawalHash: BytesLike, index: number = 0): Promise<FinalizeWithdrawalParams>
+```
+
+#### Example
+
+```ts
+import { Wallet, Provider, utils } from "zksync-ethers";
+import { ethers } from "ethers";
+
+const PRIVATE_KEY = "<WALLET_PRIVATE_KEY>";
+
+const provider = Provider.getDefaultProvider(types.Network.Sepolia);
+const ethProvider = ethers.getDefaultProvider("sepolia");
+const wallet = new Wallet(PRIVATE_KEY, provider, ethProvider);
+
+const WITHDRAWAL_HASH = "<WITHDRAWAL_TX_HASH>";
+const params = await wallet.finalizeWithdrawalParams(WITHDRAWAL_HASH);
+```
+
+### `requestExecute`
+
+Request execution of L2 transaction from L1.
+
+#### Inputs
+
+| Name                             | Type                      | Description                                                                                                                                                                                                                            |
+| -------------------------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `transaction.contractAddress`    | `Address`                 | The L2 contract to be called (optional).                                                                                                                                                                                               |
+| `transaction.calldata`           | `BytesLike`               | The input of the L2 transaction (optional).                                                                                                                                                                                            |
```

Comment:
nit: Should the `Type` be `string`, `string<DataHexString>`, or `DataHexString` here? Or do we support `Bytes` in the function?

Reply:
`BytesLike` is an alias for `ArrayLike<number> | string`, so both `'0x...'`, and and `ethers.utils.arrayify('0x...')` works.